doc/whats_new/v0.21.rst

```diff
@@ -177,6 +177,11 @@ Support for Python 3.4 and below has been officially dropped.
   with the document and the caller functions.
   :issue:`6463` by :user:`movelikeriver <movelikeriver>`.
 
+- |Fix| :func:`ensemble._gradient_boosting._partial_dependence_tree` now takes
```

Comment:
please reference the public functions rather than private methods.

Reply:
As #12599 is deprecating `ensemble.partial_dependence` should I anticipate and refer to `inspect.partial_dependence` ?

Reply:
You can use `ensemble`, I'll edit it in #12599  to indicate that they've been deprecated.

Reply:
Thanks, done !